### DISCUSSION TITLE
Run postgrest in development mode

### DISCUSSION THREAD

**Discussion by nilooy** [PostgREST/postgrest#2407] [Q&A] [answered]
I'm new to `Haskell`. I need help understanding how to run `postgrest` locally with a watch mode.

### Steps taken
1. I forked and cloned the repo
2. ran `stack build`

but what my next step should be?  Can anyone help please?

**Comment by steve-chavez** [comment, marked as answer]
Our Nix tooling has all you need for development! See the guide https://github.com/PostgREST/postgrest/blob/main/nix/README.md

There's a watch command already included

```
$ nix-shell

$ postgrest-watch -h
postgrest-watch
Usage: postgrest-watch [-h|--help] [--] <command> ...
        <command>: Command to run
        ... : command arguments
        -h, --help: Prints help

Watch the project for changes and reinvoke the given command.

Example:
  postgrest-watch postgrest-test-io
```

**Reply by nilooy** [reply]
Thanks a lot for the quick response. 😊
